index.d.ts

```diff
@@ -339,6 +339,32 @@ export class TimeoutError extends Error {
 	constructor();
 }
 
+/**
+A `Symbol` that can be returned by a `beforeRetry` hook to stop the retry.
+This will also short circuit the remaining `beforeRetry` hooks.
+
+@example
+```
+import ky, {stop as kyStop} from 'ky';
```

Comment:
I'd name it `RETRY_STOP`.

Reply:
We went with something generic so we can use the symbol for other places we want to support stopping.

Reply:
Okay, then it should be `SYMBOL_STOP` or `symbol_stop` IMO :P

Reply:
Stop is too general I think

Reply:
I don't really like hungarian notation. Prefixing it with "symbol" doesn't make the intent any clearer.

Reply:
@szmarczak thankfully that will generate a `TypeError`. But I get your point. I have the same icky feelings about objects with plural names. I tend to reserve plural names for arrays and it can be confusing if there is no distinction in some APIs. Symbols add to the problem because they are often used to signal intent for an action, similar to a function, so the naming styles conflict. TypeScript and IntelliSense make this a bit more obvious, though.